test/app/books/register/addBookDiv.test.tsx

```diff
@@ -1,14 +1,15 @@
 import { fireEvent, render } from '@testing-library/react'
 import { user1 } from '../../../__utils__/data/user'
 
-const addBookActionMock = jest.fn()
-jest.mock('@/app/books/register/actions', () => ({
-  __esModule: true,
-  addBook: { bind: () => () => addBookActionMock() },
-}))
+describe('add book div component', async () => {
+  const { addBookActionMock } = vi.hoisted(() => {
+    return { addBookActionMock: vi.fn() }
+  })
+  vi.mock('@/app/books/register/actions', () => ({
+    addBook: { bind: () => addBookActionMock },
+  }))
 
-describe('add book div component', () => {
-  const AddBookDivComponent = require('@/app/books/register/addBookDiv').default
+  const AddBookDivComponent = (await import('@/app/books/register/addBookDiv')).default
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg)
vi.mockについて[ドキュメント](https://vitest.dev/api/vi.html#vi-mock)に以下のように記載されているので、

> The call to vi.mock is hoisted, so it doesn't matter where you call it. It will always be executed before all imports. 
（vi.mockの呼び出しはホイストされているので、どこで呼び出しても構わない。常にすべてのインポートの前に実行される。）

vi.mock でモックしていれば await import ではなく 通常の import でも問題なかったりするかも？と思いました
（jestだとホイストされなかったので `require` を使っている認識のため）

Reply:
対応しました 2c7082d